docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+version: '3.8'
```

Comment:
per previous conversations, I'd like to remove these separate ideas around local dev improvements to speed up the approval and review of the base image upgrades. We can have a separate PR and discussions to talk through ideas like this